axonserver/src/main/java/io/axoniq/axonserver/localstorage/LocalEventStore.java

```diff
@@ -68,15 +68,19 @@
     private long newPermitsTimeout=120000;
 
     private final int maxEventCount;
+    private final int blacklistedSendAfter;
```

Comment:
Does this parameter state the number of blacklisted event _types_ or the number of blacklisted _events_?
I think which of both is the case should be made clear, through JavaDoc for example.